frontend/src/utils/offlineMessages.js

```diff
@@ -3,19 +3,27 @@
  */
 module.exports = {
   offline: {
-    de_DE: {
-      OFFLINE_MESSAGE_LINE1: 'Verbindung unterbrochen',
-      OFFLINE_MESSAGE_LINE2:
-        'Es treten Verbindungsprobleme auf. Bitte kontrolliere die Verbindung und lade die Seite neu.',
-      BAD_GATEWAY_LINE1: 'Instanz ist nicht verfügbar',
-      BAD_GATEWAY_LINE2: 'Es treten Verbindungsprobleme auf.',
+    de: {
```

Comment:
consider moving this part in some offline_de.json file.

Reply:
https://github.com/metasfresh/metasfresh/pull/12082#discussion_r764737620